pkg/netflow/common/constants.go

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package common
+
+const (
+	// DefaultStopTimeout is the default stop timeout
+	DefaultStopTimeout = 5
+
+	// DefaultAggregatorFlushInterval is the default flush interval
+	DefaultAggregatorFlushInterval = 10
+
+	// DefaultAggregatorBufferSize is the default aggregator buffer size interval
+	DefaultAggregatorBufferSize = 100
+)
+
+// Default ports
+const (
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏The code defines a set of constants for the ports and another set of constants for the flow types. Would it be more clear/convenient to declare specific structs for Flows so that it embeds relevant informations in a single place? It would be straightforward to add new flows (I am thinking about JFlow for Juniper). 
Something like:

```
type FlowType struct {
   name string
   defaultPort int16
...
}

var (
   TypeIPFIX = &FlowType {
      name: "ipfix",
      defaultPort:  uint16(4739)
   },
   ...
)
```